modules/nf-core/vizgenpostprocessing/preparesegmentation/main.nf

```diff
@@ -0,0 +1,60 @@
+process VIZGENPOSTPROCESSING_PREPARESEGMENTATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    container 'ghcr.io/wehi-soda-hub/vizgen-postprocessing_container:v0.1.1'
+
+    input:
+    val(meta)
+    path(algorithm_json)
+    path(input_images)
+    val(images_regex)
+    path(um_to_mosaic_file)
+    val(tile_size)
+    val(tile_overlap)
+
+    output:
+    tuple val(meta), path("*.json"), path(input_images), path(algorithm_json), emit: segmentation_files
```

Comment:
We prefer in nf-core to only output the produced files, as the input channels are normally already present.
```suggestion
    tuple val(meta), path("*.json"), emit: segmentation_files
```